### ISSUE TITLE
rclonesync placing lsl files directly in home directory

### ISSUE THREAD

**Issue by b0o** [cjnaz/rclonesync-V2#39]
```
$ find $HOME -maxdepth 1 -name '.rclonesync*'
./.rclonesyncwd
./.rclonesyncwdLSL_dropbox-maddy__Documents__home_maddy_Dropbox__Path2_ERROR
./.rclonesyncwdLSL_dropbox-maddy__Documents__home_maddy_Dropbox__Path1_ERROR
./.rclonesyncwdLSL_dropbox-maddy__Documents__home_maddy_Dropbox__Path1
./.rclonesyncwdLSL_dropbox-maddy__Documents__home_maddy_Dropbox__Path2
```

I am assuming these files are intended to be inside of `.rclonesyncwd`. 

There seems to be a missing `/` here:
https://github.com/cjnaz/rclonesync-V2/blob/22d0ec9de63a0af16b514000e1cbd848d5515158/rclonesync.py#L80

**Comment by cjnaz** [comment]
Hum. They use to be.  :>  Running the tests overrides the default workdir.  The tests pass, and I didn't check the non-test-mode output, which also passes.  I'll fix that.  Thanks.

**Comment by cjnaz** [comment]
Fixed with release 2.9.1.
